Python/ceval.c

```diff
@@ -120,14 +120,23 @@ static size_t opcache_global_hits = 0;
 static size_t opcache_global_misses = 0;
 #endif
 
+
+static int
+thread_can_handle_signals(void)
```

Comment:
The following seems a bit more specifically descriptive of what it's doing, and might make more sense if it's ever used in the future for something other than signal handling:

```suggestion
current_is_main_thread(void)
```

Reply:
Even if only the main thread of the main interpreter can handle signals, I prefer "thread_can_handle_signals" name to make the function intent more explicit.